api/pkg/auth/google_auth.go

```diff
@@ -61,7 +61,7 @@ func InitGoogleClient(ctx context.Context, audience string) (*http.Client, error
 	}
 
 	if f.Type == serviceAccountKey {
-		return idtoken.NewClient(ctx, audience)
+		return idtoken.NewClient(ctx, "api.caraml")
```

Comment:
Nit: Would be nice declare this as a constant, at the top of this file, even if it's not expected to be explicitly set by other components (in which case, it can be a private const).

Reply:
Ahh yep right, I've just moved it up as a constant :)